_posts/2021-05-17-changelog-147.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #147"
+date: 2021-05-17 12:00:00 +0300
+---
+
+
+## New Features
+
+* [#7140] Enable [new name resolution engine](https://blog.jetbrains.com/clion/2020/12/intellij-rust-updates-for-2020-3/#new-resolve) by default
+
+* [#6456] Add WSL support. To use it, enable `org.rust.wsl` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features)
+
+* [#7102], [#7165] Annotate unknown cfg attributes and provide a quick-fix. This corresponds to the [E0537](https://doc.rust-lang.org/error-index.html#E0537) compiler error. (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-147/unknown_cfg_predicate" w="700px" %}
+
+* [#6832] Suggest target triples in command for Cargo run configuration and run anything dialog. Suggest installing the target if it is not yet installed
+
+    <img src="/assets/posts/changelog-147/suggest_target_triple.png" alt="Target triple completion in cargo run configuration" width="700px"/>
+
+* [#7098] Support basic resolve of macros 2.0 (`pub macro name() { ... }`)
+
+## Performance Improvements
+
+* [#7181] Slightly optimize cargo refresh when using new name resolution engine
+
+## Fixes
+
+* [#7174] Cancel external linter (cargo check) immediately when some file is changed
+
+* New name resolution fixes:
+
+    * [#7184] Resolve procedural macros through `#[macro_use]` on extern crates
+
+    * [#7167] Fix exception when removing crate from workspace
+
+    * [#7152] Fix changing crate name
+
+* [#7178] Take into account enum's lifetimes in `Needless Lifetimes` inspection
+
+* [#7063] Now `Can't find crate` annotation can be suppressed, highlights only unknown identifier and highlighting style is `unknown symbol` instead of general `error`
```

Comment:
```suggestion
* [#7063] Now `Can't find crate` annotation highlights only unknown identifiers and can be [suppressed](https://www.jetbrains.com/help/idea/disabling-and-enabling-inspections.html#suppress-in-editor). Also, its highlighting style was changed to `unknown symbol` (instead of `error`)
```
[minor] maybe rephrase it to something like this, in order to improve readability